content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Verifying image signatures
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+We keep adding support for verifying the integrity and authenticity of contents using the Sigstore project. In this
+post, we focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [blog](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Sigstore signatures for images that match the name provided. If all signature validation pass or
+there is no container that matches the image name, the Pod will be accepted.
+
+This policy also mutates matching images to add the image digest, therefore the version of the deployed image can't
```

Comment:
Do we mean to convey that the policy mutates/alters matching images by adding the image digest so that the version of the deployed image can't change and we have a record?
```suggestion
This policy also mutates matching images to add the image digest, therefore the version of the deployed image can't
```

Reply:
I would like to explain why mutation is needed. "Once the image has been verified with a signature, one needs to ensure that the specific image tag from that signature is the one instantiated. This is achieved by the policy by mutating the Pod request and pinning that validated tag".

Reply:
done in https://github.com/kubewarden/kubewarden.io/pull/87/commits/940ae906d54e4917ebb6aa9e7d1c5bc600cb66ed